### ISSUE TITLE
Missing jemalloc dependency

### ISSUE THREAD

**Issue by akoshchiy** [duckdb/duckdb-java#63]
Seems that duckdb sources is not bundled with jemalloc. https://github.com/duckdb/duckdb/discussions/13075
It could be the reason of some memory issues, e.g. https://github.com/duckdb/duckdb/issues/9712

**Comment by brianwyka** [comment]
@lnkuiper , do you think this could be the issue?

**Comment by lnkuiper** [comment]
@akoshchiy, indeed, the JDBC client has not been bundled with jemalloc since we moved it out of the main repository to this repository. I don't see why it can't be bundled with jemalloc, so we should look into this.

@brianwyka, I'm not sure. I have been able to reproduce a similar memory retention issue, but only on MacOS regardless of whether jemalloc was bundled. On Linux, I was not able to reproduce this issue.

**Comment by brianwyka** [comment]
@lnkuiper , do you have instructions on how to bundle it?  Perhaps I can contribute if its not on the roadmap or radar.

**Comment by lnkuiper** [comment]
You can enable jemalloc by adding it to the list of extensions in `scripts/vendor.py` and calling `python3 scripts/vendor.py --duckdb=/path/to/duckdb/repo`, and then `make release`.

For other clients, we only bundle jemalloc on x86 linux, and we'd like to keep it the same for JDBC.

**Comment by brianwyka** [comment]
@lnkuiper , finally got around to this! ☝️

**Comment by guizmaii** [comment]
Will this be fixed in DuckDB 1.2.0? 👀

**Comment by brianwyka** [comment]
@lnkuiper , I've confirmed that the `jemalloc` extensions is now available from DuckDB Java bundled in version `1.2.0`.  However, it is not loaded, only installed.  According to https://duckdb.org/docs/extensions/jemalloc.html, it can not be loaded at runtime.  IS there additional work required to have it loaded by default or am I missing something?

**Comment by staticlibs** [comment]
@brianwyka 

Just FYI, I can confirm that `jemalloc` is loaded by default and is actually used in the latest dev version on Linux.

**Comment by staticlibs** [comment]
jemalloc was added in #133 and #171, closing this as completed.
